cypress.config.ts

```diff
@@ -52,7 +52,7 @@ export default defineConfig({
 					return ip
 				})
 				.then(waitOnNextcloud)
-				.then(configureNextcloud)
+				.then(() => configureNextcloud())
```

Comment:
Not necessary :)

Reply:
Required for Typescript because void is not assignable to undefined (at runtime both are the same as a function without return value is returning undefined, but for Typescript undefined and void are not the same...).

The alternative would be
```ts
.then(waitOnNextcloud as (ip: string) => Promise<undefined>)
.then(configureNextcloud)
```

This is the error if nothing is changed:
![image](https://github.com/nextcloud/nextcloud-cypress/assets/1855448/f2d82979-d05f-44d5-b308-9c0f34c8925e)